rasa/shared/nlu/constants.py

```diff
@@ -10,9 +10,13 @@
 METADATA = "metadata"
 METADATA_INTENT = "intent"
 METADATA_EXAMPLE = "example"
+VOCABULARY = "vocabulary"
```

Comment:
Do you think this will be required by downstream dependencies in the future? I'd rather go for `rasa.nlu.constants`